src/hotspot/share/gc/z/zVerify.cpp

```diff
@@ -111,6 +111,16 @@ static bool z_is_null_relaxed(zpointer o) {
   return (untype(o) & ~color_mask) == 0;
 }
 
+static void z_verify_oop_object(zaddress addr, zpointer o, void* p) {
+  const oop obj = cast_to_oop(addr);
+  guarantee(oopDesc::is_oop(obj), BAD_OOP_ARG(o, p));
+}
+
+static void z_verify_root_oop_object(zaddress o, void* p) {
+  const oop obj = cast_to_oop(o);
+  guarantee(oopDesc::is_oop(obj), BAD_OOP_ARG(o, p));
+}
```

Comment:
I pre-reviewed this part, but I realize now that I'd like to update the parameter name for the zaddress. Would you mind updating the code this?
```suggestion
static void z_verify_root_oop_object(zaddress addr, void* p) {
  const oop obj = cast_to_oop(addr);
  guarantee(oopDesc::is_oop(obj), BAD_OOP_ARG(addr, p));
}
```